website/templates/konfig.php

```diff
@@ -221,37 +221,32 @@
 define('CONFIG_PASS', $config['pass']);
 define('CONFIG_DB', $config['db']);
 define('CONFIG_CHARSET', $config['charset']);
+define('CONFIG_TIMEZONE', $config['timezone']);
+
+// TODO // STATE FOR prevent WARNING
+// Keep only fatal, no more warning
```

Comment:
:+1: You 're right we should really prevent errors to be displayed to end user, however, as a developer it's nice to have them shown in my private instance.

Reply:
I've added check to disable warning on prod. Not 100% sure that relying on `$_SERVER['SERVER_NAME']` is the best way to do that. 
Maybe dedicated `konfig-local` entry is better?

Reply:
We could check using the technique and change it later if not reliable :)